vitest.config.ts

```diff
@@ -8,7 +8,7 @@ export default defineConfig({
   resolve: {
     alias: {
       '#build/nuxt.config.mjs': resolve('./test/mocks/nuxt-config'),
-      '#build/paths.mjs': resolve('./test/mocks/paths'),
+      '#paths': resolve('./test/mocks/paths'),
```

Comment:
i'm sure nitro at least will keep using `#internal/nitro/` scoped virtuas but it _might_ conflict with any other alias by modules we could use `#nuxt-paths`, `#app-paths`, `#app/paths` or something non conflicting. totally up to you.

Reply:
maybe `#internal/nuxt/paths`? what do you think?